packages/plugin-client-drizzle/test/integration.ava.ts

```diff
@@ -0,0 +1,2187 @@
+
+  t.deepEqual(result, [
+    { id: 1, name: 'John 0', verified: true },
+    { id: 2, name: 'John 1', verified: true },
+    { id: 3, name: 'John 2', verified: true },
+    { id: 4, name: 'John 3', verified: true },
+    { id: 5, name: 'John 4', verified: true },
+    { id: 6, name: 'John 5', verified: true },
+    { id: 7, name: 'John 6', verified: true },
+    { id: 8, name: 'John 7', verified: true },
+    { id: 9, name: 'John 8', verified: true },
+    { id: 10, name: 'John 9', verified: true }
+  ]);
+});
+
+test.skip('prepared statement with placeholder in .where', async (t) => {
+  const { db } = t.context;
+
+  await db.insert(usersTable).values({ name: 'John' });
+  const stmt = db
+    .select({
+      id: usersTable.id,
+      name: usersTable.name
+    })
+    .from(usersTable)
+    .where(eq(usersTable.id, placeholder('id')))
+    .prepare('stmt3');
+  const result = await stmt.execute({ id: 1 });
+
+  t.deepEqual(result, [{ id: 1, name: 'John' }]);
+});
+
+test.skip('prepared statement with placeholder in .limit', async (t) => {
+  const { db } = t.context;
+
+  await db.insert(usersTable).values({ name: 'John' });
+  const stmt = db
+    .select({
+      id: usersTable.id,
+      name: usersTable.name
+    })
+    .from(usersTable)
+    .where(eq(usersTable.id, placeholder('id')))
+    .limit(placeholder('limit'))
+    .prepare('stmt_limit');
+
+  const result = await stmt.execute({ id: 1, limit: 1 });
+
+  t.deepEqual(result, [{ id: 1, name: 'John' }]);
+  t.is(result.length, 1);
+});
+
+test.skip('prepared statement with placeholder in .offset', async (t) => {
+  const { db } = t.context;
+
+  await db.insert(usersTable).values([{ name: 'John' }, { name: 'John1' }]);
+  const stmt = db
+    .select({
+      id: usersTable.id,
+      name: usersTable.name
+    })
+    .from(usersTable)
+    .offset(placeholder('offset'))
+    .prepare('stmt_offset');
+
+  const result = await stmt.execute({ offset: 1 });
+
+  t.deepEqual(result, [{ id: 2, name: 'John1' }]);
+});
+
+// TODO change tests to new structure
+test.skip('migrator', async (t) => {
+  const { db } = t.context;
+
+  await db.execute(sql`drop table if exists all_columns`);
+  await db.execute(sql`drop table if exists users12`);
+  await db.execute(sql`drop table if exists "drizzle"."__drizzle_migrations"`);
+
+  await migrate(db, { migrationsFolder: './drizzle2/pg' });
+
+  await db.insert(usersMigratorTable).values({ name: 'John', email: 'email' });
+
+  const result = await db.select().from(usersMigratorTable);
+
+  t.deepEqual(result, [{ id: 1, name: 'John', email: 'email' }]);
+
+  await db.execute(sql`drop table all_columns`);
+  await db.execute(sql`drop table users12`);
+  await db.execute(sql`drop table "drizzle"."__drizzle_migrations"`);
+});
+
+test.skip('insert via db.execute + select via db.execute', async (t) => {
+  const { db } = t.context;
+
+  await db.execute(sql`insert into ${usersTable} (${name(usersTable.name.name)}) values (${'John'})`);
```

Comment:
`insert into users("name") values('John');` supposed to work. That is the statement that is executed, correct?